### PR TITLE
Rf] rename table user to user

### PR DESCRIPTION
**Pull request by bizianov**

### CODE REVIEW

File: .idea/libraries/Maven__com_fasterxml_classmate_1_1_0.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
```
Comment (AndreiLoyevets) [inline .idea/libraries/Maven__com_fasterxml_classmate_1_1_0.xml:1]: Let's try to remove .idea and target from github.
(do this in a separate branch)
1. Create a separate folder for another copy of your project.
2. git clone into selected directory.
3. Without IDEA, delete .idea and target folders.
4. Commit and push changes.
5. Check that the folders dissappeared from github.
6. Merge to develop branch.

File: crm.iml [hunk 2]
```diff
@@ -47,12 +47,27 @@
     <orderEntry type="library" name="Maven: org.springframework.data:spring-data-commons:1.11.4.RELEASE" level="project" />
```
Comment (AndreiLoyevets) [inline crm.iml:47]: this file should be also removed from github

File: .idea/libraries/Maven__org_springframework_boot_spring_boot_starter_validation_1_3_5_RELEASE.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
```
Comment (AndreiLoyevets) [inline .idea/libraries/Maven__org_springframework_boot_spring_boot_starter_validation_1_3_5_RELEASE.xml:1]: exclude this file as well together with root directory

File: pom.xml [hunk 4]
```diff
@@ -40,6 +40,11 @@
             <version>1.4.1.RELEASE</version>
         </dependency>
 
+        <dependency>
```
Comment (AndreiLoyevets) [inline pom.xml:43]: <parent> section should be at the top of pom.xml

File: pom.xml [hunk 5]
```diff
@@ -40,6 +40,11 @@
             <version>1.4.1.RELEASE</version>
```
Comment (AndreiLoyevets) [inline pom.xml:40]: Cleanup mess with versions:
parent 1.3.5.RELEASE, jpa - 1.4.1.RELEASE, web - no version at all

File: src/main/java/Application.java [hunk 6]
```diff
@@ -17,10 +14,7 @@
 @EntityScan(basePackages = {"model"})
```
Comment (AndreiLoyevets) [inline src/main/java/Application.java:14]: Read about @SprintBootApplication annotation, maybe, it is your case... Anyways, will be useful

File: src/main/java/model/User.java [hunk 7]
```diff
@@ -6,20 +6,15 @@
  * Created by slava23 on 10/11/2016.
  */
 @Entity
-@Table(name = "user")
 public class User {
 
     @Id
-    @Column(name = "id")
```
Comment (AndreiLoyevets) [inline src/main/java/model/User.java]: @Column annotation still should be present but without name

File: src/main/java/service/UserService.java [hunk 8]
```diff
@@ -4,17 +4,21 @@
...
 import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
 
 /**
  * Created by slava23 on 10/11/2016.
  */
 
 @Service
+@RestController
```
Comment (AndreiLoyevets) [inline src/main/java/service/UserService.java:15]: I guess it is changed in another branch

File: src/main/java/Application.java [hunk 9]
```diff
@@ -1,10 +1,7 @@
 import org.springframework.boot.orm.jpa.EntityScan;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
-import service.UserService;
-import model.User;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
```
Comment (AndreiLoyevets) [inline src/main/java/Application.java:4]: Ctrl+Alt+O - optimize imports

File: src/main/java/service/UserService.java (hunk 8 above)
Comment (bizianov) [reply src/main/java/service/UserService.java:15]: yes, we removed it at all

File: src/main/java/model/User.java (hunk 7 above)
Comment (bizianov) [reply src/main/java/model/User.java]: discussed in skype

File: src/main/java/Application.java (hunk 6 above)
Comment (bizianov) [reply src/main/java/Application.java:14]: removed this annotation

File: src/main/java/Application.java (hunk 9 above)
Comment (bizianov) [reply src/main/java/Application.java:4]: done

File: pom.xml (hunk 5 above)
Comment (bizianov) [reply pom.xml:40]: fixed already in another branch

File: .idea/libraries/Maven__com_fasterxml_classmate_1_1_0.xml (hunk 1 above)
Comment (bizianov) [reply .idea/libraries/Maven__com_fasterxml_classmate_1_1_0.xml:1]: will create another branch once we merge everything else

File: crm.iml (hunk 2 above)
Comment (bizianov) [reply crm.iml:47]: related to the previous comment

File: .idea/libraries/Maven__org_springframework_boot_spring_boot_starter_validation_1_3_5_RELEASE.xml (hunk 3 above)
Comment (bizianov) [reply .idea/libraries/Maven__org_springframework_boot_spring_boot_starter_validation_1_3_5_RELEASE.xml:1]: related to the previous comment

**Pull request merged**